file.c

```diff
@@ -1590,15 +1589,20 @@ rb_access(VALUE fname, int mode)
  * Document-method: directory?
  *
  * call-seq:
- *   File.directory?(file_name)   ->  true or false
+ *   File.directory?(path) -> true or false
  *
- * Returns <code>true</code> if the named file is a directory,
- * or a symlink that points at a directory, and <code>false</code>
- * otherwise.
+ * With string +object+ given, returns +true+ if +path+ is a string path
+ * leading to a directory, or to a symbolic link to a directory; +false+ otherwise:
```

Comment:
can we add a note about `File.directory?` will return `false` when the path doesn't exist or not accessible. Not sure if the line `File.directory?('foo') # => false` implies that.

Reply:
I think "+false+ otherwise" covers all cases.`